packages/api/src/__tests__/e2e/mapi/mapi.test.e2e.ts

```diff
@@ -77,33 +71,31 @@ describe("MAPI E2E Tests", () => {
     return await fhirApi.readResource("Patient", patientId, fhirHeaders);
   };
 
-  // const getCwOrg = async (org: { oid: string }) => {
-  //   if (!isCwEnabled) return undefined;
-  //   return await cwCommands.organization.get(org.oid);
-  // };
+  it("gets settings", async () => {
+    const settings = await medicalApi.getSettings();
+    expect(settings).toBeTruthy();
+  });
 
-  // const getCqOrg = async (org: { oid: string }) => {
-  //   if (!isCqEnabled) return undefined;
-  //   return await getCqOrganization(org.oid);
-  // };
+  it("updates settings", async () => {
+    const whUrl = whServer.getWebhookServerUrl();
+    const updateResp = await medicalApi.updateSettings(whUrl);
+    expect(updateResp).toBeTruthy();
+    expect(updateResp.webhookUrl).toEqual(whUrl);
+    expect(updateResp.webhookKey).toBeTruthy();
+    const settings = await medicalApi.getSettings();
+    expect(settings).toBeTruthy();
+    expect(settings.webhookUrl).toBeTruthy();
+    expect(settings.webhookUrl).toEqual(whUrl);
+    whServer.storeWebhookKey(updateResp.webhookKey);
```

Comment:
do we want to check here too that the wh server gets a ping message since thats the other part of the behavior when calling update?

Reply:
would need to add the getter and setter stuff for pings the same way we have it for handleConsolidated

Reply:
Good call!

Added those to the 3rd PR of the group: https://github.com/metriport/metriport/pull/2228